lib/Page.js

```diff
@@ -722,9 +722,14 @@ class Page extends EventEmitter {
       } else {
         element.value = values.shift();
       }
-
-      element.dispatchEvent(new Event('change'));
```

Comment:
does this work instead of using `initEvent`?

```js
new Event("change", {"bubbles":true, "cancelable":true});
```

(they default to false). It does probably make sense for them to be true here. Good call.

Reply:
man, it sure did. I had the option syntax wrong. Any reason why bubbles and cancelable shouldn't be set to true by default?

Reply:
Nice catch @ChristianDavis!